CHANGELOG.md

```diff
@@ -34,4 +34,5 @@
 
 * Added the ability for AfterPay to be available in the selected countries.
 * Added the AfterPay localization for Netherlands, German and French language.
-* Klarna Pay Later : Remove fields gender and birthday from checkout form and customer object.
\ No newline at end of file
+* Klarna Pay Later : Remove fields gender and birthday from checkout form and customer object.
+* Removed old ginger-PHP library environment and transport to composer autoload.
```

Comment:
Let's change this explaining on something like :

Replaced locally stored ginger-php library on composer library installer.